### PR TITLE
chore: download A32NX experimental from GitHub

### PR DESCRIPTION
**Pull request by FoxtrotSierra6829**
## Summary of Changes

- move A32NX experimental version to GitHub rather than downloading it from Bunny CDN

## Additional context

Saves money
requires https://github.com/flybywiresim/a32nx/pull/7154 to be merged

### CODE REVIEW

File: src/renderer/data.ts [hunk 1]
```diff
@@ -90,8 +90,10 @@ export const defaultConfiguration: Configuration = {
                         {
                             name: 'Experimental',
                             key: 'experimental',
-                            url: 'https://cdn.flybywiresim.com/addons/a32nx/experimental',
-                            alternativeUrls: [],
+                            url: 'https://github.com/flybywiresim/a32nx/releases/download/assets/experimental/',
+                            alternativeUrls: [
+                                'https://cdn.flybywiresim.com/addons/a32nx/experimental'
```
Comment (tracernz) [inline src/renderer/data.ts:95]: Should have a trailing comma? At least the standard FBW eslint config requires that...

Comment (FoxtrotSierra6829) [reply src/renderer/data.ts:95]: The linting config here doesn't, but I can add it if you want to

Comment (tracernz) [reply src/renderer/data.ts]: Never mind. It doesn't use https://github.com/flybywiresim/eslint-config-flybywire

Comment (FoxtrotSierra6829) [reply src/renderer/data.ts]: #381

**Review by tracernz — APPROVED** [review]
🚀

**Pull request merged**